modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsControllerEpics.scala

```diff
@@ -440,7 +485,7 @@ object TcsControllerEpics extends TcsController {
     oiwfs: GuiderConfig,
     telescopeGuideConfig: TelescopeGuideConfig,
     aoFold: AoFold,
-    scienceFoldPosition: ScienceFoldPosition,
+    scienceFoldPosition: Option[ScienceFold],
```

Comment:
`Option` because the value read for the science fold position could be "invalid". If it is `None`, the science fold will be configured.

Reply:
Is "Invalid" the value in the epics channel?

Reply:
Actually, some positions have the value "undefined". And if the current position does not match any position in the LUT, the value is "INVALID". The important point is that the value is read to decide if it necessary to configure it. If the value is unknown, then it doesn't matter, we just need to configure it to the requested value.